### ISSUE TITLE
Warn Prompt: "no two valid MoneroPulse DNS checkpoint records were received"

### ISSUE THREAD

**Issue by BigslimVdub** [aeonix/aeon#55]
```
[P2P7]  WARN    net.dns src/common/dns_utils.cpp:508   WARNING: no two valid MoneroPulse DNS checkpoint records were received
```
I get the error 2 times a day:
 01:17:12.503
and
13:17:11.355

12 hours apart on the dot. 

Using 0.12.5 release on Ubuntu 16.04 server. 

Whats going on here? Per search, it looks like this is for Monero related daemon not Aeon: [Reddit link from fluffy](https://monero.stackexchange.com/questions/2508/what-is-a-p2p0warning-no-two-valid-moneropulse-dns-checkpoint-records-were-r)

Daemon seems to be working fine else wise. 

```
Height: 996700/996700 (100.0%) on mainnet, not mining, net hash 13.67 MH/s, v7, up to date, 8(out)+147(in) connections, uptime 2d 16h 19m 50s
```

**Comment by stoffu** [comment]
Monero maintains some information on their DNS TXT records which are accessed by the daemon and the wallet software. Aeon currently doesn't maintain such a mechanism, and I forgot to disable it during the rebase. You can just ignore the message.

#56

**Comment by BigslimVdub** [comment]
Was this merged with v0.12.6? If so just close it

**Comment by stoffu** [comment]
Yes.

**Comment by BigslimVdub** [comment]
Fix merged with 12.6 release
